build.gradle

```diff
@@ -289,7 +289,6 @@ configurations.all {
 }
 
 dependencies {
-    implementation 'jakarta.annotation:jakarta.annotation-api:1.3.5'
```

Comment:
This will be re-introduced in another PR once a new minimum snapshot distribution of core is available.